packages/base/src/formbuilder/objectform/process/dissolveProcessForm.tsx

```diff
@@ -85,7 +86,7 @@ export class DissolveForm extends BaseForm {
           properties: {
             ...prevState.schema?.properties,
             dissolveField: {
-              ...prevState.schema?.properties?.dissolveField,
+              ...(prevState.schema?.properties?.dissolveField as RJSFSchema),
```

Comment:
A type assertion isn't the ideal I was going for, but it's a step in the right direction, and I didn't want to spend too long on this right now :)